mmv1/third_party/terraform/services/storage/resource_storage_bucket_object.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
copyright headers aren't necessary since these are added automatically when provider is generated

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/10038/commits/e06c803fde4de09549f2e1a678e1c06f005b502d